src/status_im/subs/wallet/send.cljs

```diff
@@ -13,3 +13,17 @@
  :wallet/wallet-send
  :<- [:wallet/ui]
  :-> :send)
+
+(rf/reg-sub
+ :wallet/send-transaction-ids
+ :<- [:wallet/wallet-send]
+ :-> :transaction-ids)
+
+(rf/reg-sub
+ :wallet/send-transaction-progress
+ :<- [:wallet/send-transaction-ids]
+ :<- [:wallet/transactions]
+ (fn [[tx-ids transactions]]
+   (let [send-tx-ids (set (keys transactions))]
+     (select-keys transactions
+                  (for [k tx-ids :when (send-tx-ids k)] k)))))
```

Comment:
I think this `for` is behaving like a `filter`:

Consider using `(filter send-tx-ids tx-ids)`

Reply:
hmm, will have to reconsider what exactly is being done but I think I didn't use filter for a particular reason. will check again

Reply:
checked and updated 👍